playbooks/provisioning/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -4,7 +4,7 @@ public_dns_domain: "example.com"
 public_dns_nameservers: []
 
 openstack_ssh_public_key: "openshift"
-openstack_default_image_name: "rhel73"
+openstack_default_image_name: "centos7"
 openstack_default_flavor: "m1.medium"
```

Comment:
nit: it seems we may need also, or some subset of it, for CI cases:
``openshift_disable_check: disk_availability,memory_availability,package_availability,docker_image_availability,docker_storage,package_version``